### PR TITLE
Add release note for 1.1.0 in NEWS.md

### PR DESCRIPTION
**Pull request by hcho3**
@dmlc/xgboost-committer

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -3,6 +3,206 @@ XGBoost Change Log
...
+* GPU-side data sketching now works with data from external memory (#5365).
+
+### Parameter validation: detection of unused or incorrect parameters (#5477, #5569, #5508)
+* Mis-spelled training parameter is a common user mistake. In previous versions of XGBoost, mis-spelled parameters were silently ignored. Starting with 1.0.0 release, XGBoost will produce a warning message if there is any unused training parameters. The 1.1.0 release makes parameter validation available to the scikit-learn interface (#5477) and the R binding (#5569).
+
+### Thread-safe, in-place prediction method (#5389, #5512)
+* Previously, the prediction method was not thread-safe (#5339). This release adds a new API function `inplace_predict()` that is thread-safe. It is now possible to serve concurrent requests for prediction using a shared model object.
```
Comment (trivialfis) [inline NEWS.md:25]: Cudf too.

File: NEWS.md [hunk 2]
```diff
@@ -3,6 +3,206 @@ XGBoost Change Log
...
+* Avoid rabit calls in learner configuration (#5581)
+* Hide C++ symbols in libxgboost.so when building Python wheel (#5590). This fixes apache/incubator-tvm#4953.
+* Fix compilation on Mac OSX High Sierra (10.13) (#5597)
+* Fix build on big endian CPUs (#5617)
+* Resolve crash due to use of `vector<bool>::iterator` (#5642)
+* Validation JSON model dump using JSON schema (#5660)
+
+### Performance improvements
+* Wide dataset quantile performance improvement (#5306)
+* Use `scikit-learn` in extra dependencies (#5310)
```
Comment (trivialfis) [inline NEWS.md:140]: Why is this a perf improvement?

File: NEWS.md (hunk 1 above)
Comment (hcho3) [reply NEWS.md:25]: Added.

File: NEWS.md (hunk 2 above)
Comment (hcho3) [reply NEWS.md:140]: My bad. Should be under build system.

**Pull request merged**